### PR TITLE
NotificationsWhenBusy

### PR DESCRIPTION
**Pull request by DoubleStyx**
Allows you to receive notifications while your status is set to Busy.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -622,6 +622,29 @@
...
+                "DoubleStyx": {
+                    "url": "https://github.com/DoubleStyx"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/DoubleStyx/NotificationsWhenBusy/releases/tag/1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/DoubleStyx/NotificationsWhenBusy/releases/download/1.0.0/NotificationsWhenBusy.dll",
```
Comment (EIA485) [inline manifest.json:640]: leads to a 404, seems to because the release tag is v1.0.0 not 1.0.0

**Comment by EIA485** [comment]
slightly confused about a comment in the latest source, the binary seems to be built from this commit [e46c218
]( https://github.com/DoubleStyx/NotificationsWhenBusy/blob/e46c218dc9b1bd7d4a7e4a93023fc5bae384ab34/NotificationsWhenBusy/NotificationsWhenBusy.cs) however in the latest source there is [this](https://github.com/DoubleStyx/NotificationsWhenBusy/blob/master/NotificationsWhenBusy/NotificationsWhenBusy.cs#L25) comment stating that the config system is not working

**Review by ljoonal — CHANGES_REQUESTED** [review]
Like @EIA485 noted (tho without pressing the request changes button 😛), it's a 404.

**Comment by DoubleStyx** [comment]
It’s not intentional; probably a result of me creating a project on a Razer laptop. I’ll have to scrutinize some assembly properties a bit more next time I build.

**Pull request merged**